### ISSUE TITLE
 helm.elastic.co is missing 7.13.3 charts

### ISSUE THREAD

**Issue by arosenheinrich** [elastic/helm-charts#1295]
helm.elastic.co is missing charts for the 7.13.3 versions released 5 days ago.

**Comment by rgarcia89** [comment]
I can confirm
```
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
elastic/elasticsearch   7.13.2          7.13.2          Official Elastic helm chart for Elasticsearch
elastic/elasticsearch   7.13.1          7.13.1          Official Elastic helm chart for Elasticsearch
elastic/elasticsearch   7.13.0          7.13.0          Official Elastic helm chart for Elasticsearch
```

**Comment by bzuppard** [comment]
I can also confirm...
`Error: chart "elasticsearch" version "7.13.3" not found in https://helm.elastic.co repository`

**Comment by ebuildy** [comment]
What is the deploy process? Most of github projet use github action https://github.com/helm/chart-releaser-action , work pretty well.

**Comment by robertmarsal** [comment]
Until the new version is released you can work around it by setting the imageTag: `imageTag=7.13.3`

**Comment by rgarcia89** [comment]
@robertmarsal this is not going to bring you the updates in the kubernetes manifests that became part of the 7.13.3 version

**Comment by robertmarsal** [comment]
that's true, for some reason I understood that OP wanted to use the latest ES version not the chart. The change I suggested only accomplishes that

**Comment by rgarcia89** [comment]
@jmlrt are you aware of that?

**Comment by jmlrt** [comment]
Hi, Thanks for reporting this. 
After some investigations, It seems we had an issue with 7.13.3 tag which was created on the wrong branch.
This is fixed now and 7.13.3 charts should be available soon.

**Comment by rgarcia89** [comment]
awesome chart is there now
thanks a lot
```
# helm search repo elastic/elasticsearch --versions
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
elastic/elasticsearch   7.13.3          7.13.3          Official Elastic helm chart for Elasticsearch
```
